eks-hybrid-iam-ra-setup.sh

```diff
@@ -51,6 +70,18 @@ cat << EOF > "${SECRETS_DIR}/node.key"
 ${NODE_KEY_DATA}
 EOF
 
+if ! [ "${DRY_RUN}" = "true" ]; then
```

Comment:
You can probably do an early return here:
```suggestion
if [ "${DRY_RUN}" = "true" ]; then
  exit 0
fi
```

Then out-dent the signing helper service stuff. This will also suppress the `apiclient` commands, which seems in keeping with the spirit of a dry-run mode.

Reply:
Done